tools/webpack-jumpstart

```diff
@@ -100,6 +100,10 @@ if test -n "${partial}"; then
     # fetch the available commits, then iterate our history looking for the first match
     available="$(git_cache ls-remote origin 'refs/tags/sha-*' | cut -f2 -d-)"
     for commit in $(git log -n 100 --format=%H); do
+        if git show --oneline --name-only "$commit" | grep -q '^node_modules'; then
```

Comment:
I think there's an off-by-one error here: if we have a jumpstart build for this commit (ie: the commit that made the change, ie: directly after the change) then we should use it.

So put this check below the "if available ..." below, right?

Reply:
Whoops, you are right of course. Fixed.